apps/nextjs-website/src/app/layout.tsx

```diff
@@ -61,19 +78,21 @@ export default async function RootLayout({
         )}
       </head>
       <ThemeRegistry options={{ key: 'mui' }}>
-        <body>
-          {environment === 'prod' && (
-            <div
-              key='script-cookie'
-              dangerouslySetInnerHTML={{ __html: COOKIE_SCRIPT }}
-            ></div>
-          )}
-          <AuthProvider>
-            <SiteHeader products={products} />
-            <MainWrapper>{children}</MainWrapper>
-            <SiteFooter />
-          </AuthProvider>
-        </body>
+        <NextIntlClientProvider locale={'it'} messages={messages}>
+          <body>
+            {environment === 'prod' && (
```

Comment:
Let's start reducing code duplication: now you can use the `isProduction` available [here](https://github.com/pagopa/developer-portal/blob/046d4aaaa49809ffccd8b7f21f7f0ec5d4551b57/apps/nextjs-website/src/config.ts#L6)
```suggestion
            {isProduction && (
```

Reply:
If you accept this suggestion from here, you have to add the import manually 😄